index.html

```diff
@@ -1,15 +1,61 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Velzck Portfolio V2.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
     <link rel="stylesheet" href="style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Crete+Round&family=Poppins&display=swap"
+        rel="stylesheet">
     <title>Portfolio </title>
 </head>
+
 <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header>
+
+        <div id="header" class="flexColumn">
+            <div id="headerNav" class="headerCont roboto">
+                <p id="hour">VelzckC0D3</p>
+                <img id="menu" src="style/img/menu.svg" alt="">
+            </div>
+        </div>
+
+    </header>
+
+    <main class="flexColumn">
+        <section id="main" class="flexColumn">
+            <div id="mainCont">
+                <h1>Hey There. I'm Alejandro</h1>
+                <h2 class="roboto">I'm a Software Developer</h2>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div id="mainIcons">
+                    <a href="https://github.com/VelzckC0D3/" target="_blank" rel="noopener" aria-label="Github"><img class="icon" src="style/icons/github.svg"
+                            alt="GitHub"></a>
+                    <a href="https://www.linkedin.com/in/velzckcode/" target="_blank" rel="noopener" aria-label="Linkedin"> <img class="icon" src="style/icons/linkedin.svg"
+                            alt="Linkedin"></a>
+                    <a href="https://www.angellist.com/" target="_blank" rel="noopener" aria-label="AngelList"> <img class="icon" src="style/icons/angelList.svg"
+                            alt="AngelList"></a>
+                    <a href="https://twitter.com/velzck/" target="_blank" rel="noopener" aria-label="Twitter"> <img class="icon" src="style/icons/twitter.svg"
+                            alt="Twitter"></a>
+                    <a href="https://medium.com/" target="_blank" rel="noopener" aria-label="Medium"> <img class="icon" src="style/icons/medium.svg" alt=""></a>
+
+                </div>
+            </div>
```

Comment:
The list of icons needs to have a `ul tag` to group them, and `li tag` per icon; each icon has a link (a tag).

Apart from being semantically correct, the `a tag` will let you connect your social networks with the icons, using `href attribute`.

Reply:
Done! 
![image](https://user-images.githubusercontent.com/92229666/219451647-1d324f87-c20b-4730-8e42-dd109f00e3ae.png)

Reply:
Also, added on the PR the specifications of everything that I had added, as you requested! 
![image](https://user-images.githubusercontent.com/92229666/219452914-4fba0aa2-308a-4fe0-bc58-225455c94092.png)